libraries/AP_GPS/GPS_Backend.cpp

```diff
@@ -353,8 +353,20 @@ bool AP_GPS_Backend::calculate_moving_base_yaw(const float reported_heading_deg,
 
 #ifndef HAL_BUILD_AP_PERIPH
         {
-            const Vector3f antenna_tilt = AP::ahrs().get_rotation_body_to_ned() * offset;
+            // get lag
+            float lag = 0.1;
+            get_lag(lag);
```

Comment:
```suggestion
            float lag;
            if (!get_lag(lag)) {
                lag = 0.1;
            }
```

A small nitpick, but I'd like to mage get_lag a warn if unused, so it's just one less place to clean up

Reply:
I'd actually prefer to make get_lag() not be bool and just return a best guess float. The consumers of get_lag() are not going to stop what they are doing when get_lag() fails (imagine if we stopped running the EKF if get_lag decided to return false). The lag is a compensation factor.
A bool return implies that the caller will take a different code path when it fails. I can't think of any example where we would want to do that. Also, the code that is best placed to make a best guess at a lag if one isn't really known is AP_GPS, so it should be the one making the guess. So a float return makes most sense, and makes for simpler code